packages/vsce/README.md

```diff
@@ -212,14 +212,6 @@ View resource information without using the CICS tree. Access the command palett
 <img src="/packages/vsce/docs/images/resource-inspector.gif" alt="CICS Resource Inspector" width="800px"/>
 </p>
 
-### Show attributes
```

Comment:
Removing this block will prevent a user getting confused and mixed messages - do we want to replace this with a block about Resource Inspector? I'll tag in @davenice and @jt-nti who have been thinking about removing a lot of the detailed feature explanations from the Readme. 

Perhaps removing this is enough for this change, in favour of a cutdown readme in the future?

Reply:
I think there does need to be a new Resource Inspector section to replace the outgoing Show attributes section.

If we do decide to cut down the readme, the content will go somewhere and I don't think abandoning it in the mean time is the right thing to do.